src/pages/profiles/ProfileDetailHeader.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC, useState } from "react";
+import { Link } from "react-router-dom";
+import DeleteProfileBtn from "./actions/DeleteProfileBtn";
+import { LxdProfile } from "types/profile";
+import ProfileRename from "pages/profiles/forms/ProfileRename";
+import { Tooltip } from "@canonical/react-components";
+
+interface Props {
+  name: string;
+  profile?: LxdProfile;
+  project: string;
+}
+
+const ProfileDetailHeader: FC<Props> = ({ name, profile, project }) => {
+  const [isRename, setRename] = useState(false);
+  const canRename = profile?.name !== "default";
```

Comment:
Nothing really bad would happen since the parent of the element where this is used will not be rendered when profile is not provided, but I see your point. I updated it for the sake of code clarity/cleanliness.